plugins/source/test/resources/services/test_some_table.go

```diff
@@ -36,13 +37,19 @@ func TestSomeTable() *schema.Table {
 				PrimaryKey:  true,
 			},
 		},
+		Relations: schema.Tables{
+			testSubTable(),
+		},
 	}
 }
 
 func fetchSomeTableData(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
-	res <- map[string]any{
-		"column1": "test_project_id",
-		"column2": 123,
+	cl := meta.(*client.Client)
+	for i := 0; i < *cl.Spec.NumRows; i++ {
+		res <- map[string]any{
+			"resource_id": i + 1,
+			"column2":     fmt.Sprintf("test_column_data_%d", i%3),
```

Comment:
why `%3`?

Reply:
Simple counter, but also duplicates data to some extent.